packages/components/addon/components/hds/table/index.hbs

```diff
@@ -0,0 +1,42 @@
+<table class={{this.classNames}} ...attributes>
+  {{#if this.isSortableTable}}
+    <caption class="sr-only" aria-live="polite">{{@caption}} {{this.sortedMessageText}}</caption>
+  {{else}}
+    <caption class="sr-only">{{@caption}}</caption>
+  {{/if}}
+  <thead class="hds-table__thead">
+    {{#if this.isSortableTable}}
+      <Hds::Table::Tr>
+        {{#each this.columns as |column|}}
+          {{#if column.isSortableColumn}}
+            <Hds::Table::ThSort
+              @sortKey={{column.key}}
+              @sortBy={{this.sortBy}}
+              @sortOrder={{this.sortOrder}}
+              @onClick={{this.setSortBy}}
+            >
+              {{column.label}}
+            </Hds::Table::ThSort>
+          {{else}}
+            <Hds::Table::Th>{{column.label}}</Hds::Table::Th>
+          {{/if}}
+        {{/each}}
+      </Hds::Table::Tr>
+    {{else}}
+      {{yield (hash Tr=(component "hds/table/tr") Th=(component "hds/table/th")) to="head"}}
```

Comment:
I don't actually think this works properly because you can't do something like 

```
<Hds::Table as |T|>
  <:head>
    <T.Tr>
      <T.Th>Header One</T.Th>
     </T.Tr>
   <:/head>
 </Hds::Table>
 ```
 
 but I'm also wondering why it's working when I manually invoke the longhand form of the component if I've told it to yield the shorthand. 🤔